pixi.toml

```diff
@@ -4,9 +4,12 @@ version = "0.1.0"
 description = "Reader to load data from DICOM files into Apache Arrow"
 authors = ["Marc Garcia <garcia.marc@gmail.com>"]
 channels = ["conda-forge"]
-platforms = ["linux-64"]
+platforms = ["linux-64", "osx-arm64"]  # --> add in "osx-arm64" for MacOS
 
 [tasks]
-main = "LD_LIBRARY_PATH=target/debug python dicom_reader/__init__.py"
+main = "DYLD_LIBRARY_PATH=target/debug python dicom_reader/__init__.py"   
+
+##For MacOS change from LD_LIBRARY_PATH --> DYLD_LIBRARY_PATH 
 
 [dependencies]
+python = "3.11.4.*"
```

Comment:
Probably better to use `3.11.*`. When `3.11.5` is released, our library should still be compatible, since patch versions shouldn't change backward compatibility, and it would be better to use, since it'll contain security fixes.

Reply:
Done